### PR TITLE
Add support for solc-0.5.8

### PR DESCRIPTION
**Pull request by xwvvvvwx**
Also added instructions for updating solc versions

### CODE REVIEW

**Comment by xwvvvvwx** [comment]
closes https://github.com/dapphub/dapptools/issues/183

File: README.md [hunk 1]
```diff
@@ -78,8 +78,7 @@ unreleased versions.
 
 NOTE: not all versions are supported on macOS platforms.
 
-## TODO
+### Maintainance
```
Comment (asymmetric) [inline README.md]: maintenance

Comment (xwvvvvwx) [reply README.md]: thx, done.

**Comment by xwvvvvwx** [comment]
build failed on macOS: https://travis-ci.com/dapphub/dapptools/jobs/199001100#L1746

Do we have a mac I could debug on?

**Comment by asymmetric** [comment]
Found [this PR](https://github.com/NixOS/nixpkgs/pull/60539).

**Comment by asymmetric** [comment]
Upstream has merged 0.5.8, we should use that instead of our fork.

**Comment by xwvvvvwx** [comment]
@asymmetric done. Also moved  `0.5.6` and `0.5.7` to released.

File: README.md [hunk 2]
```diff
@@ -78,8 +78,7 @@ unreleased versions.
 
 NOTE: not all versions are supported on macOS platforms.
 
-## TODO
+### Maintenance
```
Comment (asymmetric) [inline README.md]:
```suggestion
### Contributing
```

File: src/dapp/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.20.0] - 2019-05-23
```
Comment (asymmetric) [inline src/dapp/CHANGELOG.md]: We need a link at the bottom

Comment (xwvvvvwx) [reply src/dapp/CHANGELOG.md]: what do you mean?

Comment (asymmetric) [reply src/dapp/CHANGELOG.md]: At the bottom of the file.

Comment (xwvvvvwx) [reply src/dapp/CHANGELOG.md]: ah I see. Done now.

**Pull request merged**